prjxray/grid.py

```diff
@@ -1,7 +1,20 @@
 from collections import namedtuple
+import enum
+from prjxray import segment_map
+
+
+class SegmentType(enum.Enum):
```

Comment:
These aren't segment types. Segment type has a very specific meaning in tilegrid. Rather these are "Block Type". See UG470 pg 110: https://www.xilinx.com/support/documentation/user_guides/ug470_7Series_Config.pdf

Reply:
I was having trouble naming this enum, I'll update it.